app/services/pii/cacher.rb

```diff
@@ -32,15 +32,26 @@ def fetch
     end
 
     def fetch_string
-      user_session[:decrypted_pii]
```

Comment:
Can you help me understanding why the logic below is necessary? It looks like `SessionEncryptor#kms_encrypt_pii!` _should_ prevent `user_session[:encrypted_pii]` from being an issue.

Reply:
Slacked with Mitchell and see where I got hung up. The session encryptor _encrypts_ the `user_session[:encrypted_pii]` value, but never actually decrypts it. It leaves this to the session encryptor so we don't need to run KMS on every request for a proofed user.

Reply:
Attempted to clarify this better in cfea392bd4ea2e8e87b5fa70cabc0d1294f80dd2